react/components/CustomTags.tsx

```diff
@@ -187,7 +189,8 @@ export const CustomRenderers = {
   ),
   paragraph: (props: any) => {
     if (props.children === '' || props.children.length === 0) return null
-    return <p className="t-body c-on-base mt0 lh-copy mb6">{props.children}</p>
+
+    return checkDocProp(props) ? (<p className="t-body c-on-base mt0 lh-copy mb6">{props.children}</p>) : getDocProp(props)
```

Comment:
Is it the right logic? I mean if it has the docprop placeholder, it should get `getDocProp` instead of returning the `p` tag, no?

Reply:
yes, i accidentally switched it